src/styles/app.css

```diff
@@ -0,0 +1,12 @@
+.App {
+  display: flex;
+  flex-direction: column;
+  background-color: navy;
+}
+
+.nasa-logo {
+  height: 200pt;
+  width: 350pt;
```

Comment:
Interesting choice of units here! `pt` values are only supposed to be used for text values, they are a bit of an anachronism from what I understand as they are based on a subdivision of an inch. 
Would recommend sticking to `px` units here